pkg/cloud/aws/services/ec2/vpc.go

```diff
@@ -124,6 +128,12 @@ func (s *Service) describeVPC(clusterName string, id string) (*v1alpha1.VPC, err
 		return nil, NewConflict(errors.Errorf("found more than one vpc with supplied filters. Please clean up extra VPCs: %s", out.GoString()))
 	}
 
+	switch *out.Vpcs[0].State {
+	case ec2.VpcStateAvailable, ec2.VpcStatePending:
```

Comment:
Is this needed if we are filtering on the VPC states we are checking for here?

Reply:
I guess we don't just added it to be explicit in case we ever change the filters